tests/framework/results/test_manager.py

```diff
@@ -224,3 +224,17 @@ def test_add_observation_nop_stratifications(
     )
     for m in match:
         assert m in caplog.text
+
+
+def test_setting_default_stratifications(mocker):
```

Comment:
This is fine. Do we already have a test that tests that `set_default_stratifications` works as expected?

Reply:
Not directly - I'll add one.